photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/estimators/kernels/Matern52.scala

```diff
@@ -35,17 +35,19 @@ import breeze.stats.stddev
  * @see "Practical Bayesian Optimization of Machine Learning Algorithms" (PBO),
  *   https://papers.nips.cc/paper/4522-practical-bayesian-optimization-of-machine-learning-algorithms.pdf
  *
+ * @param indexedTransformMap the map specifies the indices and transformation function of hyper-parameters
  * @param amplitude the covariance amplitude
  * @param noise the observation noise
  * @param lengthScale the length scale of the kernel. This controls the complexity of the kernel, or the degree to which
  *   it can vary within a given region of the function's domain. Higher values allow less variation, and lower values
  *   allow more.
  */
 class Matern52(
+    indexedTransformMap: Map[Int, Double => Double] = Map(),
```

Comment:
Minor: let's move this to the end of the argument list in all places. You'd expect to see the kernel parameters first when scanning this.